### DISCUSSION TITLE
SunshineGameLauncher

### DISCUSSION THREAD

**Discussion by klattimer** [LizardByte/community#165] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

I tried a few suggested methods of getting around the launching of epic/steam games with sunshine. I've settled on a small Visual Studio project to get it working. 

I wanted to notify you of this project, and maybe create a search trail for people who want the same thing. Check the code here, it's unfinished and raw but it took an hour and I'm happy it mostly works. 

https://github.com/klattimer/SunshineGameLauncher

### Host Operating System

Windows

### Operating System Version

win 10

### Architecture

64 bit

### Sunshine commit or version

latest release

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

RTX3070

### GPU Driver/Mesa Version

latest

### Config

```Shell
not necessary.
```

### Relevant log output

```shell
Sorry for spam :)
```

**Comment by cooljimy84** [comment]
I was wondering why my epic and steam games launched, but moonlight closed saying it had failed. Then when i connected to the desktop found the game running waiting for me.

**Reply by cooljimy84** [reply]
So from my messing around it seems that the PID that sunshine launches closes/changes causing sunshine to disconnect /close thinking the app has closed ?
as if i have batch file with the following it works as a "fire and forget command"
```
steam.exe -applaunch 320140
ping 0.0.0.0 -n 10
```
This stalls the close of the PID long enough for the stream to start.

**Comment by ReenigneArcher** [comment]
If this is something you're maintaining, feel free to add it to https://github.com/LizardByte/awesome-sunshine

[a comment by an automated account is not shown]
